compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/AnnotationDesugar.java

```diff
@@ -884,15 +977,36 @@ private BLangInvocation getInvocation(BLangLambdaFunction lambdaFunction) {
     }
 
     private int calculateIndex(List<BLangStatement> statements, BTypeSymbol symbol) {
+        return calculateOCEExprIndex(statements, symbol);
+//        for (int i = 0; i < statements.size(); i++) {
```

Comment:
Clean up this in https://github.com/ballerina-platform/ballerina-lang/issues/34395